### PR TITLE
Moved goto into the tablet (currently not enabled)

### PR DESCRIPTION
**Pull request by druiz17**
got goto to working the tablet, waiting on the new UI design to finish the implementation

### CODE REVIEW

**Comment by druiz17** [comment]
Test: no function changes but ensure that goto still works

**Comment by hifi-gustavo** [comment]
Build console output for commit: 00a540ec4e24c91b15c1d94b07b24ad1bb5c52f2 available at: https://gist.github.com/72806b03619ca9e499ff790ea47bfe01<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9635-00a540e.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9635-00a540e.dmg

**Comment by claudioshg** [comment]
testing

**Comment by claudioshg** [comment]
### Desktop PC - Windows 10 Home - CPU 4.0GHz Intel i7 - GTX1070 - RAM 16GB - Tested with Oculus Rift and in Desktop Mode

Tested: Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9635-00a540e.exe

PASS
- Tested GOTO in Oculus VR and in Desktop Mode. 
- No function changes and GOTO works as before. 

NOTES:
Everything works. Switched domains effortlessly, and all the functions of the window work as well.

File: interface/resources/qml/hifi/Card.qml [hunk 1]
```diff
@@ -129,7 +129,7 @@ Rectangle {
     property int dropSamples: 9;
     property int dropSpread: 0;
     DropShadow {
-        visible: showPlace && desktop.gradientsSupported;
+        visible: showPlace; //&& desktop.gradientsSupported;
```
Comment (sethalves) [inline interface/resources/qml/hifi/Card.qml]: Is this change going to have some unintended consequence?

Comment (druiz17) [inline interface/resources/qml/hifi/Card.qml]: I am not sure

Comment (sethalves) [inline interface/resources/qml/hifi/Card.qml]: I asked Howard about it -- apparently drop shadows require "transparency/gradients, which doesn't work on Mac. Or at least, didn't".  I guess we should see what this looks like on osx.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 66d8796bbb29d6942c5432aec5f338bf93b3aca1 available at: https://gist.github.com/72806b03619ca9e499ff790ea47bfe01<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9635-66d8796.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9635-66d8796.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0e83e5da0654606458756a0bb4a0b88bd1b52f38 available at: https://gist.github.com/72806b03619ca9e499ff790ea47bfe01<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9635-0e83e5d.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9635-0e83e5d.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ff60058cbe219a59d55286975f48b62a0d570fc3 available at: https://gist.github.com/72806b03619ca9e499ff790ea47bfe01<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9635-ff60058.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9635-ff60058.dmg

**Pull request merged**